ksql-version-metrics-client/src/main/avro/KsqlVersionMetrics.avsc

```diff
@@ -21,6 +21,11 @@
       "name": "ksqlComponentType",
       "type": "string",
       "doc": "Identifies the ksql component"
+    },
+    {
+      "name": "isActive",
+      "type": "boolean",
+      "doc": "Indicates if the server was active in the last hour"
```

Comment:
Can we document what 'active' actually means here, e.g. I'm assuming its something to do with it having received a request with in the last hour.

Reply:
In the design doc we had the following as the definition of activeness:

`A KSQL server will send an "active" ping (i.e., the activeness data field is set to "yes" or "active") if, in between the time of the last ping and the next, upcoming ping, it has executed >= 1 KSQL query (CS/CT/CSAS/CTAS/SELECT) or statement (SHOW TABLES etc.).  Otherwise, an "idle" ping will be sent.`

I agree that it needs to also consider the current live queries even though the `CSAS/CTAS` statement has been received and executed before the last ping time. 
I'll update the definition in the doc and also the implementation in the PR to consider the existence of live queries too.